flexibee-core/src/main/java/com/adleritech/flexibee/core/api/domain/PaymentBinding.java

```diff
@@ -0,0 +1,21 @@
+package com.adleritech.flexibee.core.api.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.simpleframework.xml.Element;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class PaymentBinding {
```

Comment:
Flexibee nazev je "uhrada", ale preklad je "settlement", to se mi moc nelibi. Ale rad pouziju lepsi nazev.